core/src/main/java/com/novoda/noplayer/internal/exoplayer/forwarder/BufferStateForwarder.java

```diff
@@ -26,32 +26,37 @@ public void onPlayerStateChanged(boolean playWhenReady, int playbackState) {
     }
 
     @Override
+    public void onRepeatModeChanged(@com.google.android.exoplayer2.Player.RepeatMode int repeatMode) {
```

Comment:
No, I can't. I don't understand how a static import can work here. The Class name matches in both cases so it won't be able to distinguish between them.